Wireframe/css/style.css

```diff
@@ -0,0 +1,163 @@
+/* Reset default margin for body element */
+body {
+  margin: 0;
+  background-color: #b4b4b4;
+}
+
+/* Default font styles for body and button elements */
+body, button {
+  font-family: "Source Sans Pro", "Helvetica Neue", Arial, sans-serif;
+  color: #292b2c;
+  font-size: 1rem;
+}
+
+/* Reset margins for heading elements */
+h1, h2, h3, h4, h5, h6 {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+/* Reset margins for paragraphs */
+p {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+/* Header section styles */
+.site-header {
+  display: block;
+  text-align: center;
+  padding: 20px;
+  text-shadow: 2px 2px 4px rgba(255, 255, 255, 0.5);
+  color: #70184b;
+}
+
+/* Main content styles */
+main {
+  max-width: 800px;
+  margin: 0 auto;
+  padding: 2rem;
+}
+
+.main-container {
+  background-color: #f2f2f2;
+  padding: 20px;
+  border: 1px solid #0a0a0a;
+  height: auto;
+  border-radius: 5px;
+  position: relative;
+  font-size: 17px;
+  line-height: 1.5;
+  text-align: justify;
+  margin-bottom: 10px;
+}
+
+/* Main image styling */
+.img-pic {
+  background-size: 100% 100%; 
+  border-bottom: 1px solid #0a0a0a;
+  border-radius: 10px 5px 0 0;
+  width: 798px;
+  height: 400px;
+  position: relative;
+  line-height: 1.5;
+  text-align: justify;
+  margin: -20px;
+  margin-bottom: 10px;
+}
+
+/* Box styling */
+.box {
+  padding: 0;
+  width: 390px;
+  height: 230px;
```

Comment:
What would happen if your content was more than 230px?

Take a look at this CSS Grid course, it can help you write flexible layouts. As a web developer, you never really know what size screen your site is shown on, so it's a good idea to always write in a flexible style.

https://scrimba.com/playlist/pWqLHa

Reply:
I found out that it wont show and sometimes its outside the box effecting other elements in the page 

I am going to implement all of the things  you told me about in which I'm grateful for. 
thank you for your time and your superb comments